azuredevops/internal/acceptancetests/resource_check_exclusive_lock_test.go

```diff
@@ -0,0 +1,60 @@
+//go:build (all || resource_check_exclusive_lock) && !exclude_approvalsandchecks
+// +build all resource_check_exclusive_lock
+// +build !exclude_approvalsandchecks
+
+package acceptancetests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/acceptancetests/testutils"
+)
+
+func TestAccCheckExclusiveLock_basic(t *testing.T) {
+	projectName := testutils.GenerateResourceName()
+	timeout := 43200
+	newTimeout := 50000
```

Comment:
`timeout` is between 1~ 43200

Reply:
I changed it to `21600` in the `newTimeout`